.ci/jenkins/Jenkinsfile.deploy

```diff
@@ -55,7 +56,8 @@ pipeline {
                     }
 
                     // Verify we have the connectivity to Openshift
-                    openshift.loginOpenshift()
+                    // TODO: Uncomment this line when we have the Openshift credentials
```

Comment:
What do we need to get them? Or is this in progress in any way?

Reply:
AFAIK the OpenShift Instance is only used to run the tests. I'm not sure who has the credentials but we need to figure that out to add those credentials to Apache Jenkins.
@porcelli Do you know who can help us with that?

Reply:
we don't have a OpenShift instance available, so better disable this